src/services/History/clearHistory.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const clearHistory = async ({ videoStateDispatch }) => {
+  try {
+    const res = await axios.delete("/api/user/history/all");
+    if (res.status === 200) {
+      videoStateDispatch({ type: "HANDLE_HISTORY", payload: res.data.history });
+    }
+  } catch (e) {
+    console.log(e);
```

Comment:
Suggestion: implement some logic to let user know the issue instead of the display a message in the console.